test/integration/cluster_filter_integration_test.cc

```diff
@@ -10,32 +11,48 @@
 namespace Envoy {
 namespace {
 
+// Helper class for passing runtime information from a test filter to the test class.
+// May only be called from one filter during a test.
```

Comment:
Is it passing runtime information, or allowing for verification of the order of function calls?

Reply:
Meant the latter with the former, will rephrase to make this clear.

Reply:
Changed from
``` 
// Helper class for passing runtime information from a test filter to the test class.
```
to
```
// Helper class for passing function call order information from a test filter to the test class.
```